pyInterface/module/_config.py

```diff
@@ -171,6 +171,11 @@ def _readBinning(binningString):
 				return []
 	return retval
 
+def _roundSignificantDigits(value, significantDigits):
```

Comment:
IMHO `_roundToSignificantDigits`/`_roundToNSignificantDigits` would be a better name